Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/ExposureKeyDomain.kt

```diff
@@ -0,0 +1,44 @@
+package de.rki.coronawarnapp.submission
+
+import KeyExportFormat
+import com.google.android.gms.nearby.exposurenotification.TemporaryExposureKey
+import com.google.protobuf.ByteString
+
+class ExposureKeyDomain {
+
+    companion object {
+
+        private const val ROLLING_PERIOD = 144
+        private const val MAXIMUM_KEYS = 14
+    }
+
+    fun transformToKeyHistoryInExternalFormat(
```

Comment:
What about moving these to the constructor, so we have a 
`ExposureKeyConverter(arg1,arg2).createExternalFormat() : ReturnType`

If this is the "external api" for this classes logic, please add a specific return type, which makes it easier to understand  when viewing without IDE, i.e. currently I cant'see what the method should return.